jbpm-test/src/test/java/org/jbpm/test/ProcessMain.java

```diff
@@ -48,12 +67,103 @@ private static KieBase readKnowledgeBase() throws Exception {
 		return kbuilder.newKieBase();
 	}
 	
-	private static void startUp() {
-		JBPMHelper.startH2Server();
-		JBPMHelper.setupDataSource();
-		// please comment this line if you already have the task service running,
-		// for example when running the jbpm-installer
-		JBPMHelper.startTaskService();
+	public static void startUp() {
+	    cleanupSingletonSessionId();
+	    Properties properties = ProcessMain.getProperties();
+	    String datasourceName = properties.getProperty("datasourceName", "jdbc/jbpm-ds");
+		PersistenceUtil.setupPoolingDataSource(properties, datasourceName);
+	    String humanTaskEnabled = properties.getProperty("taskservice.enabled", "false");
+	    if ("true".equals(humanTaskEnabled)) {
+	        startTaskService();
+	    }
+	}
+
+	public
+	static void cleanupSingletonSessionId() {
+	    File tempDir = new File(System.getProperty("java.io.tmpdir"));
+	    if (tempDir.exists()) {
+	        String[] jbpmSerFiles = tempDir.list(new FilenameFilter() {
+	            @Override
+	            public boolean accept(File dir, String name) {	                
+	                return name.endsWith("-jbpmSessionId.ser");
+	            }
+	        });
+	        for (String file : jbpmSerFiles) {	            
+	            new File(tempDir, file).delete();
+	        }
+	    }
+	}
+
+	public static TaskService startTaskService() {

```

Comment:
Every method except `main` and `startUp` can be declared private here. By doing this we will be sure that it is only used in the `ProcessMain` class.